include/openthread/platform/crypto.h

```diff
@@ -610,6 +615,20 @@ otError otPlatCryptoEcdsaGenerateKey(otPlatCryptoEcdsaKeyPair *aKeyPair);
  */
 otError otPlatCryptoEcdsaGetPublicKey(const otPlatCryptoEcdsaKeyPair *aKeyPair, otPlatCryptoEcdsaPublicKey *aPublicKey);
 
+/**
+ * Import the provided key-pair into PSA.
+ *
+ * @param[in]  aKeyPair           A pointer to an ECDSA key-pair structure where the key-pair needs to be stored.
+ * @param[in]  aDerBytes          A pointer to an ECDSA key-pair in DER format.
+ * @param[in]  aDerLength         Length of the key-pair to be stored.
+ *
+ * @retval OT_ERROR_NONE          Public key was retrieved successfully, and @p aBuffer is updated.
+ * @retval OT_ERROR_PARSE         The key-pair DER format could not be parsed (invalid format).
+ * @retval OT_ERROR_INVALID_ARGS  The @p aContext is NULL.
+ *
+ */
+otError otPlatCryptoImportKeyPair(const otPlatCryptoEcdsaKeyPair *aKeyPair, uint8_t *aDerBytes, uint8_t aDerLength);
```

Comment:
Is this function supposed to save the `aDerBytes` in PSA storage?
Or it is supposed to change the `aKeyPair`? (should not since it si `const`)?

The provided weak implementation seems to copy the `aDerByte` into the `aKeyPair->mDerBytes` (do we need to check if it may be set to `null`?)

Reply:
It is supposed to import the `aDerBytes` into the `aKeyPair`. In case of PSA, it will be stored in PSA ITS instead of the array. Previoisly `mDerBytes` could never be null, as instantiating the `KeyPair` would initialise the pointer. Now that the array has been reinstated, it wont be needed as well.